src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Deleted.cshtml.cs

```diff
@@ -55,9 +59,18 @@ public IActionResult OnPostDelete(string oldUrl)
 
     private void Load()
     {
-        var items = _redirectsService.GetDeleted().ToPagedList(Paging.PageNumber, Paging.PageSize);
+        var items = FindRedirects().ToPagedList(Paging.PageNumber, Paging.PageSize);
         Message =
             $"There are currently {items.TotalItemCount} URLs that return a Deleted response. This tells crawlers to remove these URLs from their index.";
         Items = items;
     }
+    
+    private List<CustomRedirect> FindRedirects()
```

Comment:
ToPagedList might make use of IEnumerable, but now the whole list will be materialized.  This might have an impact in case of large lists.

Is there a reason not to keep it IEnumerable?

Reply:
No, let's make it IEnumerable, if you see potential problems.